backbone.js

```diff
@@ -1647,14 +1647,15 @@
       // support the `hashchange` event, HTML5 history, or the user wants
       // `hashChange` but not `pushState`.
       if (!this._hasHashChange && this._wantsHashChange && !this._usePushState) {
-        var iframe = document.createElement('iframe');
-        iframe.src = 'javascript:0';
-        iframe.style.display = 'none';
-        iframe.tabIndex = -1;
+        this.iframeElement = document.createElement('iframe');
+        this.iframeElement.src = 'javascript:0';
+        this.iframeElement.style.display = 'none';
+        this.iframeElement.tabIndex = -1;
```

Comment:
Yeah unfortunately - jsDOM dies here: https://github.com/jashkenas/backbone/pull/3613/files#diff-0d56d0d310de7ff18b3cef9c2f8f75dcR1697 as `iframe.contentFrame` is undefined :disappointed: 

The change in marionette is still required as jsDOM was throwing an error when removing an element that ain't in the DOM - so so strange. It could be because its an older version of jsDOM but we can't upgrade as they stopped supporting node.